content/docs/reference/microcontrollers/nano-33-ble.md

```diff
@@ -0,0 +1,60 @@
+---
+title: "Arduino Nano33 BLE"
+weight: 3
+---
+
+The [Arduino Nano33 BLE](h) is a very small ARM development board based on the Nordic Semiconductor [nrf52840](https://www.nordicsemi.com/eng/Products/nRF52840) processor.
+
+## Interfaces
+
+| Interface | Hardware Supported | TinyGo Support |
+| --------- | ------------- | ----- |
+| GPIO      | YES | YES |
+| UART      | YES | YES |
+| SPI      | YES | YES |
+| I2C      | YES | YES |
+| ADC      | YES | YES |
+| PWM      | YES | YES |
+
+## Machine Package Docs
+
+[Documentation for the machine package for the Arduino Nano33 IoT](../machine/arduino-nano33-ble)
+
+## Installing BOSSA
+
+In order to flash your TinyGo programs onto the Arduino Nano33 BLE board, you will need to install the "arduino_bossac" command line utility which is a special build of the [BOSSA command line utilities](https://github.com/shumatech/BOSSA).
+
+### macOS
+
+Instructions needed here.
```

Comment:
Find instructions (for all OS) here: https://github.com/tinygo-org/tinygo/blob/e02f308d43ff43fa649f4129c15e0f3809420801/src/machine/board_nano-33-ble.go#L7